supervisor/hassos.py

```diff
@@ -104,7 +104,8 @@ async def load(self) -> None:
                 raise NotImplementedError()
 
             cpe = CPE(self.sys_host.info.cpe)
-            if cpe.get_product()[0] != "hassos":
+            os_name = cpe.get_product()[0]
+            if os_name != "hassos" and os_name != "haos":
```

Comment:
```suggestion
            if os_name not in ("hassos", "haos"):
```
a bit more pythonic :)